signature-aggregator/metrics/metrics.go

```diff
@@ -31,6 +31,7 @@ var Opts = struct {
 	InvalidSignatureResponses          prometheus.CounterOpts
 	SignatureCacheHits                 prometheus.CounterOpts
 	SignatureCacheMisses               prometheus.CounterOpts
+	CachedSignatureWeightPercentage    prometheus.GaugeOpts
```

Comment:
I feel like a Prometheus Gauge is not a great fit for this metric for consumption. It doesn't describe health/state of application as a whole but of current request. I do think that we want this data available but maybe a log would be better fit. 

An idea for useful usage of gauges might be currently connected stake-weight percentage with subnets as labels.